R analysis scripts/1. validation_analysis.R

```diff
@@ -0,0 +1,623 @@
+rm(list = ls())
```

Comment:
Can you rename this to `analysis_scripts/1_validation_analysis.R` and the other files similarly? This is (a) for consistency with the rest of the pipeline, and (b) because I don't think the fact that the scripts happen to be written in R matters for describing their purpose.

Reply:
I will do this now, in order to be able to make progress.